en/docs/get-started/single-sign-on-and-identity-federation.md

```diff
@@ -0,0 +1,509 @@
+# Single Sign-On and Identity Federation
```

Comment:
This doc is not needed in the `get started` section. It should be there in `concepts`